### ISSUE TITLE
About data preparation

### ISSUE THREAD

**Issue by ljthnu** [richard-peng-xia/LMPT#6]
Thank you for your excellent work, could you please provide the class_freq.pkl file for voc and coco? Looking forward to your reply!

**Comment by richard-peng-xia** [comment]
Hi @Jintao-Li1129.

Thanks for your interest!

The `class_freq.pkl` is available at [this repo](https://github.com/wutong16/DistributionBalancedLoss). Specifically, [VOC](https://github.com/wutong16/DistributionBalancedLoss/blob/master/appendix/VOCdevkit/longtail2012/class_freq.pkl) and [COCO](https://github.com/wutong16/DistributionBalancedLoss/blob/master/appendix/coco/longtail2017/class_freq.pkl).

I hope it may help you. Please feel free to ask if there is any further problem.

**Comment by ljthnu** [comment]
Thanks, I found some errors in the **lmpt/csel.py**  while reproducing your project, class_weights is a 2D vector after processing, so `class_weights = (1 / self.class_counts) ** self.gamma / torch.sum((1 / self. class_counts) ** self.gamma,dim=2)` can't be done, please also check!

**Comment by xiapeng1110** [comment]
Hi @Jintao-Li1129,

The class weight is a 1D tensor. Please see the example mentioned [here](https://github.com/richard-peng-xia/LMPT/issues/4).

Feel free to ask if there is any further problem.
